cmd/agent/health.go

```diff
@@ -78,7 +79,12 @@ var counter = &agent.State{
 // handles pinging the endpoint and returns an error if the
 // agent is in an unhealthy state.
 func pinger(c *cli.Context) error {
-	resp, err := http.Get("http://localhost:3000/healthz")
+	healthcheckAddress := c.String("healthcheck-addr")
```

Comment:
i wonder if we could use url.Parse instead ...
and if host = "" use localhost

Reply:
I tried playing with `url.Parse`, but it can't even parse `0.0.0.0:3000` for example. So I don't think that's the solution, as it looks like it doesn't handle ports in combination with IP addresses that well.

Reply:
thanks for clarification :)